frontend/crowd-density/index.html

```
@@ -182,685 +254,2114 @@
-            <div class="story-canvas" id="state-story-canvas">
+
+            <div class="story-canvas"
+                id="state-story-canvas">
+
                 <div class="canvas-visual-frame">
-                    <img id="state-story-img" src="" alt="State Map">
-                    <div id="state-svg-container">
-                        <!-- The SVG path of the state will be injected here -->
+
+                    <img id="state-story-img"
+                        src=""
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)